city-finder.js

```diff
@@ -9,15 +9,16 @@ const getCitiesForState = (state) => {
       return getCoordinatesForCities(state, incidentCities)
     })
     .then(response => {
-      console.log(response)
+      console.log(response.map(cities => Object.values(cities)))
       svg.selectAll("circle")
           .data(
-            response.map(city => Object.values(city)).map(array => {
-              return [-array[0].longitude, array[0].latitude]})
+            response.map(cities => Object.values(cities))
+            // response.map(city => Object.values(city)).map(array => {
+            //   return [-array[0].longitude, array[0].latitude]})
           )
           .enter()
           .append("circle")
-          .attr("cx", d => d[0])
+          .attr("cx", d => projection([d.longitude, d.latitude])[0])
```

Comment:
The only clue I found- which doesn't help, is this https://github.com/d3/d3-geo/issues/39.... @brittanystoroz